File: ABC/052/A.py
import sys
A,B,C,D = map(int,input().split())
if A > 10000 or B > 10000 or C > 10000 or D > 10000:
    sys.exit()
if A * B >= C * D:
    print(A*B)
    sys.exit()
if A * B <= C*D:
    print(C*D)
    sys.exit()

File: ABC/ruby/052/A.rb
A,B,C,D = gets.chomp.split(' ')
exit if !((A =~ /^[0-9]+$/) && (B =~ /^[0-9]+$/) && (C =~ /^[0-9]+$/) && (D =~ /^[0-9]+$/))
a = A.to_i
b = B.to_i
c = C.to_i
d = D.to_i
exit if !(( 1 <= a && a <= 10**4 ) && ( 1 <= b && b <= 10**4 ) && ( 1 <= c && c <= 10**4 ) && ( 1 <= d && d <= 10**4 ))

puts (a*b >= c*d) ? a*b : c*d